swap_meet/clothing.py

```diff
@@ -1,2 +1,10 @@
-class Clothing:
-    pass
\ No newline at end of file
+from swap_meet.item import Item
+
+class Clothing(Item):
+    def __init__(self, id = None, fabric = "Unknown", condition = 0):
```

Comment:
Small stylistic reminder: According to the [Pep 8 guidelines](https://peps.python.org/pep-0008/#other-recommendations), we don't use spaces around the "=" in keyword arguments.

```suggestion
    def __init__(self, id=None, fabric="Unknown", condition=0):
```